artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -2971,6 +2975,7 @@ public void run() {
                else if (consumerRate < threshold) {
                   RemotingConnection connection = null;
                   RemotingService remotingService = ((PostOfficeImpl) postOffice).getServer().getRemotingService();
+                  ActiveMQServer server = ((PostOfficeImpl) postOffice).getServer();
```

Comment:
The code now casts twice.  Should this be in the interface? Should the casting be done once?  Can server be extracted for the line above?